hmda/Jenkinsfile

```diff
@@ -6,34 +6,31 @@ podTemplate(label: 'buildSbtPod', containers: [
 volumes: [
   hostPathVolume(mountPath: '/var/run/docker.sock', hostPath: '/var/run/docker.sock'),
 ]) {
-   node('buildSbtPod') {
-     def repo = checkout scm
-     def gitCommit = repo.GIT_COMMIT
-     def gitBranch = repo.GIT_BRANCH
-     def gitDescribe = sh(returnStdout: true, script: 'git describe --always --tags')
-     def gitTag = sh(returnStdout: true, script: "git tag --sort version:refname | tail -1").trim()
-     def commitId = sh(returnStdout: true, script: 'git rev-parse HEAD')
-     def shortCommit = commitId[0..7]
-     def result = sh(returnStatus: true, script: "git log -1 | grep '\\[deploy pr\\]'")
-     def isDeployPR = result == 0 ? true : false
-     if (gitBranch == "master") {
-       env.DOCKER_TAG = "latest"
-     }
-     else if (isDeployPR) {
-        env.DOCKER_TAG = "${env.JOB_NAME?.split('/')[1]}-${env.BUILD_NUMBER}"
-        shortCommit = env.DOCKER_TAG+shortCommit
-     }
-     else {
-       env.DOCKER_TAG = env.TAG_NAME
-     }
+  node('buildSbtPod') {
+    sh "env | sort"
 
-     println "DOCKER_TAG: ${env.DOCKER_TAG}, gitDescribe: ${gitDescribe}, gitTag: ${gitTag}, commitId: ${commitId}, isDeployPR: ${isDeployPR}"
+    def repo = checkout scm
+    def gitBranch = repo.GIT_BRANCH
+    def gitCommit = repo.GIT_COMMIT
+    def shortCommit = repo.GIT_COMMIT[0..7]
+    def isDeployPR = sh(returnStdout: true, script: "git log -1").contains("[deploy pr]")
+    def taggedBranch = env.TAG_NAME != null
+     
+    if (gitBranch == "master") {
+      env.DOCKER_TAG = "latest"
+    } else if (taggedBranch) {
+      env.DOCKER_TAG = env.TAG_NAME
+    } else {
+      env.DOCKER_TAG = env.BRANCH_NAME
+    }
+
+    println "DOCKER_TAG: ${env.DOCKER_TAG}, gitTag: ${gitTag}, commitId: ${commitId}, isDeployPR: ${isDeployPR}"
```

Comment:
Should we update this to print other information?

Reply:
Anything you think would be useful? Maybe `gitbranch`?

Reply:
Yeah, I think so. Sorry, I really just meant because I don't think `gitTag` exists anymore.

Reply:
Oh, so it doesn't. I'll update that.